evalml/pipelines/utils.py

```diff
@@ -742,9 +743,17 @@ def _make_component_list_from_actions(actions):
             cols_to_drop.extend(action.metadata["columns"])
         elif action.action_code == DataCheckActionCode.IMPUTE_COL:
             metadata = action.metadata
+            parameters = metadata.get("parameters", {})
             if metadata["is_target"]:
                 components.append(
-                    TargetImputer(impute_strategy=metadata["impute_strategy"])
+                    TargetImputer(impute_strategy=parameters["impute_strategy"])
+                )
+            else:
+                impute_strategies = parameters["impute_strategies"]
+                components.append(
+                    PerColumnImputer(
+                        impute_strategies=impute_strategies, impute_all=False
```

Comment:
With the merge of #3267, we don't need the `impute_all` parameter any more!

Reply:
That's true! Thank you @eccabay 🙏